### PR TITLE
Fix failing tests from #9250

### PR DESCRIPTION
**Pull request by mik-laj**
CC: @Khrol 

Related PR: https://github.com/apache/airflow/pull/9250

> The problem with the CI in this build was that it only tested the static tests. As no ".py" file was changed, CI skipped rest of the unit test. However on Master we run all the tests, hence Master failed with below error:
> 
> ```
> tests/www/test_views.py .............................F
> 
> _________ TestAirflowBaseViews.test_escape_in_tree_view_0_hello_world __________
> 
> a = (<tests.www.test_views.TestAirflowBaseViews testMethod=test_escape_in_tree_view_0_hello_world>,)
> 
>     @wraps(func)
>     def standalone_func(*a):
> >       return func(*(a + p.args), **p.kwargs)
> 
> /usr/local/lib/python3.6/site-packages/parameterized/parameterized.py:530: 
> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
> tests/www/test_views.py:646: in test_escape_in_tree_view
>     self.check_content_in_response(f'"conf":{{"abc":"{seralized_test_str}"}}', resp)
> tests/www/test_views.py:177: in check_content_in_response
>     self.assertIn(text, resp_html)
> ```
> 
> ```
> tests/www/test_views.py F
> 
> _________ TestAirflowBaseViews.test_escape_in_tree_view_1_hello_world __________
> 
> a = (<tests.www.test_views.TestAirflowBaseViews testMethod=test_escape_in_tree_view_1_hello_world>,)
> 
>     @wraps(func)
>     def standalone_func(*a):
> >       return func(*(a + p.args), **p.kwargs)
> 
> /usr/local/lib/python3.6/site-packages/parameterized/parameterized.py:530: 
> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
> tests/www/test_views.py:646: in test_escape_in_tree_view
>     self.check_content_in_response(f'"conf":{{"abc":"{seralized_test_str}"}}', resp)
> tests/www/test_views.py:177: in check_content_in_response
>     self.assertIn(text, resp_html)
> E   AssertionError: '"conf":{"abc":"hello\\\\u0027world"}' not found in '\n\n\n\n\n\n\n\n\n\n 
> ```
> 

---
Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [X] Target Github ISSUE in description if exists
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [X] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/www/templates/airflow/tree.html [hunk 1]
```diff
@@ -136,7 +136,7 @@
 
 var devicePixelRatio = window.devicePixelRatio || 1;
 // JSON.parse is faster for large payloads than an object literal (because the JSON grammer is simpler!)
-var data = JSON.parse({{ data|tojson }});
+var data = {{ data }};
```
Comment (Khrol) [inline airflow/www/templates/airflow/tree.html]: I think we should not remove `JSON.parse` with `tojson` due to performance problems (see comment above).

Comment (ashb) [reply airflow/www/templates/airflow/tree.html]: Correct. This was added for a reason. As it says in the comment directly before this line.

Comment (mik-laj) [reply airflow/www/templates/airflow/tree.html]: Okay. I will correct it. Thanks for your vigilance.

**Review by ashb — CHANGES_REQUESTED** [review]
Do not remove the `JSON.parse`

File: tests/www/test_views.py [hunk 2]
```diff
@@ -627,11 +627,11 @@ def test_view_uses_existing_dagbag(self, endpoint, mock_get_dag):
         self.check_content_in_response('example_bash_operator', resp)
 
     @parameterized.expand([
-        ("hello\nworld", "hello\\\\nworld"),
-        ("hello'world", "hello\\\\u0027world"),
-        ("<script>", "\\\\u003cscript\\\\u003e"),
+        ("hello\nworld", r'"conf":{"abc":"hello\nworld"}}'),
+        ("hello'world", r'"conf":{"abc":"hello\u0027world"}}'),
+        ("<script>", r'"conf":{"abc":"\u003cscript\u003e"}}'),
```
Comment (Khrol) [inline tests/www/test_views.py]: Can we add `{"\"": ""}` test here? I'm not sure if this PR still keeps the original bug fix.

**Comment by ashb** [comment]
@mik-laj @Khrol Would one of you be able to update the test runner to run tests in case of template changes too please?

File: tests/www/test_views.py (hunk 2 above)
Comment (mik-laj) [reply tests/www/test_views.py]: I added this case.

File: airflow/www/templates/airflow/tree.html (hunk 1 above)
Comment (mik-laj) [reply airflow/www/templates/airflow/tree.html]: I added JSON.parse.

**Comment by mik-laj** [comment]
> @mik-laj @Khrol Would one of you be able to update the test runner to run tests in case of template changes too please?

I'm too busy to do it now.

**Review by ashb — APPROVED** [review]

**Pull request merged**